packages/flutter_tools/lib/src/commands/assemble.dart

```diff
@@ -106,10 +114,17 @@ class AssembleCommand extends FlutterCommand {
     argParser.addOption(kExtraGenSnapshotOptions);
     argParser.addOption(kExtraFrontEndOptions);
     argParser.addOption(kDartDefines);
+    argParser.addOption(kSplitAot);
     argParser.addOption(
       'resource-pool-size',
       help: 'The maximum number of concurrent tasks the build system will run.',
     );
+    argParser.addMultiOption('target-abis',
+      splitCommas: true,
+      defaultsTo: <String>['armeabi-v7a', 'arm64-v8a', 'x86_64'],
+      allowed: <String>['armeabi-v7a', 'arm64-v8a', 'x86_64'],
+      help: 'The abis for deferred component verification.',
```

Comment:
Not clear what this means

Reply:
This was left over from previous version, need to clean it up.